compute_tools/src/compute.rs

```diff
@@ -1367,6 +1368,47 @@ LIMIT 100",
         download_size
     }
 
+    pub async fn set_role_grants(
+        &self,
+        db_name: &str,
+        schema_name: &str,
+        privileges: &[Privilege],
+        role_name: &str,
```

Comment:
NIT

```suggestion
        db_name: &PgIdent,
        schema_name: &PgIdent,
        privileges: &[Privilege],
        role_name: &PgIdent,
```
?

If it's possible and makes sense

Reply:
BTW, not sure, but maybe if passed by value, it won't require `.to_string()` below?